src/azul/terra.py

```diff
@@ -500,6 +501,11 @@ def run_sql(self, query: str) -> BigQueryRows:
                         sleep(delay)
                     else:
                         raise e
+                except InternalServerError as e:
```

Comment:
Let's also handle 503's this way

```suggestion
                except (InternalServerError, ServiceUnavailable) as e:
```